.github/workflows/_build_rosetta.yaml

```diff
@@ -65,32 +73,95 @@ jobs:
         with:
           images: ${{ env.UPLD_IMAGE }}
           flavor: latest=false
-          tags: type=raw,value=${{ github.run_id }}-${{ inputs.BASE_LIBRARY }}
+          tags: type=raw,value=${{ github.run_id }}-${{ inputs.BASE_LIBRARY }}-${{ matrix.PLATFORM }}
           labels: org.opencontainers.image.created=${{ inputs.BUILD_DATE }}
 
       - name: Set up Docker Buildx
         uses: docker/setup-buildx-action@v2
         with:
           driver-opts: |
-            image=moby/buildkit:v0.10.6
+            image=moby/buildkit:v0.12.1
 
-      - name: Pull and validate BASE_IMAGE=${{ env.BASE_IMAGE }} contains BASE_LIBRARY=${{ inputs.BASE_LIBRARY }}
+      - name: Pull and validate BASE_IMAGE=${{ steps.defaults.outputs.BASE_IMAGE }} contains BASE_LIBRARY=${{ inputs.BASE_LIBRARY }}
```

Comment:
Why do we need this step?

Reply:
Sanity mainly. I was thinking in the future with OCI annotations, we could just inspect the environment vars: https://github.com/NVIDIA/JAX-Toolbox/blob/main/.github/workflows/scripts/get_remote_env.sh and then avoid us having to pull the image; but didn't implement that yet.

Reply:
Actually I failed to mention why it's required; since (rosetta) pax does not re-install paxml, having paxml already installed in editable is required for the rosetta build at the moment. Once we have reproducible builds addressed, we can probably toss this step out.

Reply:
Since the need for the check is transient and will be eliminated soon with pip-compile, I'd recommend that we simply don't do it. The build process will also fail if the library is not present, right? It should be fine as long as the wrong library does not cause a silent failure.

Reply:
Done!